[SPARK-29531][SQL][TEST]refine ThriftServerQueryTestSuite.blackList to reuse black list in SQLQueryTestSuite

fuwhu opened the pull request:
### What changes were proposed in this pull request?
This pr refine the code in ThriftServerQueryTestSuite.blackList to reuse the black list of SQLQueryTestSuite instead of duplicating all test cases from SQLQueryTestSuite.blackList.


### Why are the changes needed?
To reduce code duplication.


### Does this PR introduce any user-facing change?
No


### How was this patch tested?
N/A

AmplabJenkins commented:
Can one of the admins verify this patch?

AmplabJenkins commented:
Can one of the admins verify this patch?

wangyum commented:
ok to test

AmplabJenkins commented:
Merged build finished. Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder-K8s/17348/
Test PASSed.

SparkQA commented:
**[Test build #112375 has started](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/112375/testReport)** for PR 26188 at commit [`605194c`](https://github.com/apache/spark/commit/605194c92035184334f543bf7e3c88f6e60cdbf8).

fuwhu committed:
refine ThriftServerQueryTestSuite.blackList to reuse black list of SQLQueryTestSuite
    by calling super.blackList instead of duplicating all test cases from SQLQueryTestSuite.blackList

SparkQA commented:
**[Test build #112375 has finished](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/112375/testReport)** for PR 26188 at commit [`605194c`](https://github.com/apache/spark/commit/605194c92035184334f543bf7e3c88f6e60cdbf8).
 * This patch passes all tests.
 * This patch merges cleanly.
 * This patch adds no public classes.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder/112375/
Test PASSed.

AmplabJenkins commented:
Merged build finished. Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder-K8s/17353/
Test PASSed.

fuwhu committed:
Add email

SparkQA commented:
**[Test build #112380 has started](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/112380/testReport)** for PR 26188 at commit [`7f8841b`](https://github.com/apache/spark/commit/7f8841b7927808632fbe1a8d4b2fa06fe9c85c4d).

AmplabJenkins commented:
Merged build finished. Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder-K8s/17354/
Test PASSed.

SparkQA commented:
**[Test build #112380 has finished](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/112380/testReport)** for PR 26188 at commit [`7f8841b`](https://github.com/apache/spark/commit/7f8841b7927808632fbe1a8d4b2fa06fe9c85c4d).
 * This patch passes all tests.
 * This patch merges cleanly.
 * This patch adds no public classes.

wangyum commented:
Merged to master
